### ISSUE TITLE
ios-deploy fails with "Failed to start debugserver: e8000022 The service is invalid" since iOS 14 beta 4

### ISSUE THREAD

**Issue by antoinevg** [trigger-corp/forge#70]
```
[  DEBUG] [100%] Installed package /Users/antoine/forge-workspace/forge-v29/development/ios/device-ios.app
[  DEBUG] ------ Debug phase ------
[  DEBUG] Starting debug of 00008020-001A09912202003A (N841AP, iPhone XR, iphoneos, arm64e) a.k.a. 'Antoine’s iPhone Xr' connected through USB...
[  DEBUG] [  0%] Looking up developer disk image
[  DEBUG] [ 95%] Developer disk image mounted successfully
[  DEBUG] Failed to start debugserver: e8000022 The service is invalid.
[  DEBUG] Waiting for the device to scan mounted image
[  DEBUG] Error 0xe8000022: The service is invalid. AMDeviceSecureStartService(device, CFSTR("com.apple.debugserver"), NULL, &con)
[  DEBUG] Failed to run /Users/antoine/forge-workspace/forge-v29/.lib/ios-deploy --debug --unbuffered --noninteractive --bundle /Users/antoine/forge-workspace/forge-v29/development/ios/device-ios.app, but was told to carry on anyway
[  DEBUG] <ForgeTask (ios)> has finished
```

**Comment by codecat** [comment]
Hi, I found this issue through Google. Noticed you closed the issue. Did you find a solution to this problem?

**Comment by antoinevg** [comment]
It was fixed upstream in:

https://github.com/ios-control/ios-deploy/commit/8092250a3c36d34d392497c1824af23d8e24b66f

I build it from source for our use so I don't know if those changes have made it to general distribution yet.

**Comment by codecat** [comment]
Awesome, thank you! I can confirm it is fixed in 1.11.2 when updating via npm, so it's actually released now. 👍
